parsers/nmparser/parse_run_details.go

```diff
@@ -80,6 +80,9 @@ func ParseRunDetails(lines []string) RunDetails {
 			runDetails.ProblemText = replaceTrim(line, "$PROB")
 		case strings.Contains(line, "#METH:"):
 			runDetails.EstimationMethods = append(runDetails.EstimationMethods, replaceTrim(line, "#METH:"))
+		case strings.HasPrefix(line, "$SIM"):
+			fields := strings.SplitN(line, ";", 2)
+			runDetails.OnlySim = strings.Contains(fields[0], "ONLYSIM")
```

Comment:
I knew that this could be the longer "ONLYSIMULATE", but I just learned it could also be shortened to "ONLY", so this will need to be reworked (will wait for other comments, though).